dlpy/applications/vgg.py

```diff
@@ -303,6 +305,9 @@ def VGG16(conn, model_table='VGG16', n_classes=1000, n_channels=3, width=224, he
         input_parameters = get_layer_options(input_layer_options, parameters)
         model.add(InputLayer(**input_parameters))
 
+        if reshape_after_input:
```

Comment:
we might also need to check its type here

Reply:
actually we should not because we explicitly specified it in the documentation but I can guarantee you someone will pass something else and then complain